content/intro-to-storybook/vue/fr/conclusion.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
+
+[📕 **Projet GitHub : chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/vue)
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
```

Comment:
@jeanphibaconnais when you're able, can you please update the link to point at https://clever-banach-415c03.netlify.app/ ?

Reply:
it's ok for this one.